depthai_demo.py

```diff
@@ -21,11 +21,22 @@
 
 sys.path.append(str(Path(__file__).parent.absolute()))
 sys.path.append(str((Path(__file__).parent / "depthai_sdk" / "src").absolute()))
-from depthai_sdk.managers import ArgsManager
+
+
+try:
+    from depthai_sdk.managers import ArgsManager
+    import cv2
+    import depthai as dai
+    import numpy as np
+except Exception as ex:
```

Comment:
@moratom shouldn't these be installed by the installer though?

Reply:
Yes, they should. But at least on my machine, the installer doesn't install them at least since version 2.0.0, while this fallback got broken more recently. I will look into why the installer doesn't install them now.